### PR TITLE
data: Glenn444

### PR DESCRIPTION
**Pull request by Glenn444**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Glenn444 [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Glenn Mark",
+  "bio": "Open Source contributor | Software engineer"
```
Comment (kumarsonsoff3) [inline data/Glenn444:3]: Please add the missing comma here. 

```suggestion
  "bio": "Open Source contributor | Software engineer",
```

File: data/Glenn444 [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      "url": "https://www.alxafrica.com/",
+      "icon": "FaLink"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/quantum_G",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
```
Comment (kumarsonsoff3) [inline data/Glenn444:17]: Also remove this trailing comma here. 

```suggestion
    }
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please add the file extension `.json` in the end your filename. Rename your file [here](https://github.com/Glenn444/BioDrop/edit/main/data/Glenn444) to `data/Glenn444.json`. 

Change `Glenn444`  ->  `Glenn444.json`

Also, make the below suggested changes.

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on BioDrop, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at t[hese Docs](https://www.biodrop.io/docs/quickstart-json) which will help with the suggested changes the maintainers have proposed. 

Please remember you can also create your Profile using [Forms](https://www.biodrop.io/docs/quickstart-forms).

**Pull request closed without merging**